docs: Add specific version install guide for `consul-k8s` CLI, refactor install (#12114)
hashicorp/consul: website/content/docs/k8s/k8s-cli.mdx

Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes.
This topic describes the commands, subcommands, and available options for using Consul K8s CLI.

## Install the Consul K8s CLI 

> The [Homebrew](https://brew.sh) package manager is required to complete the following installation instructions. Homebrew formulae will always install the latest version of a binary. If you are looking to install a specific version of the CLI please follow [Install a specific version of Consul K8s CLI](#install-a-specific-version-of-consul-k8s-cli).

Reviewer: ```suggestion
The [Homebrew](https://brew.sh) package manager is required to complete the following installation instructions. The Homebrew formulae will always install the latest version of a binary. If you are looking to install a specific version of the CLI please follow [Install a specific version of Consul K8s CLI](#install-a-specific-version-of-consul-k8s-cli).
```

   consul-k8s 0.39.0
   ```
   
## Install a specific version of Consul K8s CLI

> Since Homebrew does not provide a way to install previous versions of packages, the Consul K8s CLI will need to be installed manually. Previous versions of the Consul K8s CLI could be used to install a specific version of the Consul on Kubernetes control plane. 

Reviewer: ```suggestion
Homebrew does not provide a method to install previous versions of a package.  The Consul K8s CLI will need to be installed manually. Previous versions of the Consul K8s CLI could be used to install a specific version of Consul on the Kubernetes control plane. 
```
Cleaned up the language a bit.

1. Download the Consul K8s CLI using the following `curl` command. Enter the approriate version for your deployment via the `$VERSION` environment variable. 

>    ```shell-session
>    export VERSION=0.39.0
>    curl --location "https://releases.hashicorp.com/consul-k8s/${VERSION}/consul-k8s_${VERSION}_darwin_amd64.zip" --output consul-k8s-cli.zip

Reviewer: ```suggestion
   ```shell-session
   $ export VERSION=0.39.0
   $ curl --location "https://releases.hashicorp.com/consul-k8s/${VERSION}/consul-k8s_${VERSION}_darwin_amd64.zip" --output consul-k8s-cli.zip
```

Commands need to be prefixed with $

   ```
   
1. Unzip the zip file ouput to extract the `consul-k8s` CLI binary. This overwrites existing files and also creates a `.consul-k8s` subdirectory in your `$HOME` folder. 
   
   ```shell-session

>    unzip -o consul-k8s-cli.zip -d ~/.consul-k8s

Reviewer: ```suggestion
   $ unzip -o consul-k8s-cli.zip -d ~/.consul-k8s
```

   ```
   

> 1. Add the path to your directory. In order to persist the `$PATH` across sessions, you'll need to add this to your shellrc (i.e. shell run commands) file for the shell used by your terminal.  

Reviewer: ```suggestion
1. Add the path to your directory. In order to persist the `$PATH` across sessions, you will need to add this to your shellrc (i.e. shell run commands) file for the shell used by your terminal.  
```

   ```shell-session

>    export PATH=$PATH:$HOME/.consul-k8s/

Reviewer: ```suggestion
   $ export PATH=$PATH:$HOME/.consul-k8s/
```

   ```
   

> 1. (Optional) Issue the `consul-k8s version` command to verify the installation:

Reviewer: ```suggestion
1. (Optional) Issue the `consul-k8s version` command to verify the installation.
```
We prefer to end sentences with a period vs a semicolon, pr the HashiCorp writing style guide.

   ```shell-session

>    consul-k8s version
>    consul-k8s 0.39.0

Reviewer: ```suggestion
   $ consul-k8s version
   consul-k8s 0.39.0
```
